docs/hld.md

```diff
@@ -0,0 +1,245 @@
+            Manage -->|Connects to| ReportAPI[Report API]
+            Manage -->|Connects to| NotificationAPI
+
+            IdAMMaintenanceUI -->|Connects to| ServiceDirectoryAPI
+            IdAMMaintenanceUI -->|Connects to| IdAMAPI
+            
+            ReferralAPI -->|Calls| ServiceDirectoryAPI
+            IdAMAPI --> |Calls| ServiceDirectoryAPI
+        end
+
+        subgraph Databases
+            style ServiceDirectoryDB fill:#E2ECE9,stroke:#333 
+            style IdAMDB fill:#E2ECE9,stroke:#333 
+            style ReferralDB fill:#E2ECE9,stroke:#333 
+            style ReportDB fill:#E2ECE9,stroke:#333 
+            style NotificationDB fill:#E2ECE9,stroke:#333 
+            style ReportStagingDB fill:#E2ECE9,stroke:#333 
+
+            ServiceDirectoryAPI -->|Stores data in| ServiceDirectoryDB[s181p01-service-directory-db]
+            IdAMAPI -->|Stores data in| IdAMDB[s181p01-idam-db]
+            ReferralAPI -->|Stores data in| ReferralDB[s181p01-referral-db]
+            ReportAPI -->|Stores data in| ReportDB[s181p01-report-db]
+            NotificationAPI -->|Stores data in| NotificationDB[s181p01-notification-db]
+            ReportStagingDB[s181p01-report-staging-db]
+        end
+
+        subgraph Functions[Azure Functions]
+            style FuncOpenReferral fill:#FAD2E1,stroke:#333 
+
+            FuncOpenReferral[s181d01-fa-fh-open-referral] --> |Stores data in| ServiceDirectoryDB
+        end
+
+        subgraph DataFactory[Azure Data Factory]
+            style DataFDefault fill:#BEE1E6,stroke:#333 
+
+            DataFDefault[s181p01-dataf-default]
+            DataFDefault --> |Copies data from| ServiceDirectoryDB
+            DataFDefault --> |Copies data from| ReferralDB
+            DataFDefault --> |Copies data to| ReportStagingDB
+        end
+    end
+
+    style GovOneLogin fill:#FBF8CC,stroke:#333,stroke-dasharray: 5, 5
+    GovOneLogin[Gov One Login] -->|Authenticates| Connect
+    GovOneLogin -->|Authenticates| Manage
+
+    style LA fill:#cdf7ff,stroke:#333,stroke-dasharray: 5, 5
+    LA[Local Authority] --> |Data ingested by| FuncOpenReferral
+```
+## Service Information
+
+### Third Parties
+
+We use the following third party services:
+
+- **Postcodes.io:** Free service for postcode lookups
+- **GOV.UK One Login:** Authentication service
+- **GOV.UK Notify:** Email notification service
+
+### Services
+
+Family Hubs consists of the following services:
+
+#### UIs
+
+- **Find:** lets users find services in their area through a postcode search
+- **Connect:** allows LAs to connect families with services through connection requests
+- **Connect Dashboard:** allows LA and VS users to send/receive connection requests. Is built to be visually indistinguishable from the main Connect side, appearing as just another section of Connect.
+- **Manage:** allows LA and VCS users to manage their data, view metrics, and other administrative tasks
+- **IdAM Maintenance UI:** TODO
```

Comment:
Nitpicky but might be more in-line with the others just as `IdAM Maintenance`

Also a proposal for the `TODO`:

```suggestion
- **IdAM Maintenance UI:** allows members of the Family Hubs team to add new DfE Admin, LA and VCS users
```

Reply:
Hmmm I see what you mean, in that case I think keeping UI at the end is nicer